docker/php/Dockerfile

```diff
@@ -1,60 +1,44 @@
-FROM php:7.3-fpm
-
-ARG PHP_TIMEZONE
-ARG PHP_MEMORY_LIMIT
-ARG USER_ID
+FROM php:7.4-apache
 
+# Install packages
 RUN apt-get update && apt-get install -y \
     openssl \
     git \
     unzip \
+    libicu-dev \
     libzip-dev \
     libmagickwand-dev \
     inkscape
 
 # Install PHP extensions
-RUN docker-php-ext-install -j$(nproc) \
-        pdo \
-        pdo_mysql \
-        opcache \
-        zip
+RUN docker-php-ext-configure \
+    intl
 
-RUN apt-get install -y libicu-dev \
-    && docker-php-ext-configure intl \
-    && docker-php-ext-install intl
+RUN docker-php-ext-install -j$(nproc) \
+    intl \
+    pdo \
+    pdo_mysql \
+    opcache \
+    zip
 
-RUN pecl install \ 
-    xdebug \
```

Comment:
I would not remove `xdebug`, because for me debugging is essential for developing, but that's just my personal opinion.

Reply:
no i am sorry, i think we should keep this environment as simple as possible. xdebug is quite complex to get running if you dont have any experience (especially inside a docker container) and has a big performance penalty if configured incorrectly. i dont want to recommend (and maintain/support) this. if somebody wants to use it, he can still follow a tutorial to install it.

Reply:
I think if you are familiar enough with debugging over xdebug you will not be the typical sulu-docker user and will have your own dev environment.

Reply:
Then I would at least add an example of doing this to a `docs/` folder

Reply:
personally, i dont think thats necessary - we just want to provide a environment to quickly test sulu. i dont want to maintain a documentation how to setup xdebug inside of a docker container. this has nothing to do with sulu and you should be able to find countless tutorials for this via google 🙈

Reply:
What you describe would be the perfect setup for sulu demo. Just an environment to quickly bring up the sulu demo website, not for development. As far as I understood this repository, it should be a setup, which makes sulu development in a docker-container as **easy as possible** (and therefore should include things like xdebug and synced ssh keys). But that's just my opinion, I'm also okay with not including that. But I'd never consider using this repository, if I have to setup those things by my own, because then I have to set them up for every single project I work on. And yes, I could fork this repository and use that, but then I'd have to maintain that fork by myself, which is also not what I want (From the developers point of view)